### PR TITLE
Feanil/update vpc tools

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

File: playbooks/roles/edxapp/templates/edx-worker-lms.conf.j2 [hunk 1]
```diff
@@ -13,7 +13,7 @@ instance edx.${SERVICE_VARIANT}.core.${QUEUE}
 #env NEWRELIC={{venv_dir}}/bin/newrelic-admin
 env CONCURRENCY=${CONCURRENCY}
 env LOGLEVEL=info
-env DJANGO_SETTINGS_MODULE=lms.envs.aws
+env DJANGO_SETTINGS_MODULE=aws
```
Comment (jarv) [inline playbooks/roles/edxapp/templates/edx-worker-lms.conf.j2]: should this be a var like it is for edxapp?

File: playbooks/roles/datadog/vars/main.yml [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 datadog_api_key: "PUT_YOUR_API_KEY_HERE"
 
+datadog_apt_key: "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x226AE980C7A7DA52"
```
Comment (jarv) [inline playbooks/roles/datadog/vars/main.yml:5]: what's this change for? maybe add a comment?

Comment (feanil) [inline playbooks/roles/datadog/vars/main.yml:5]: This was changed so tha we can use the builtin apt-key ansible module so it doesn't download the key if it is already there.

**Comment by jarv** [comment]
This needs a rebase

**Comment by jarv** [comment]
ok to merge after rebase

**Pull request merged**